src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -744,6 +738,126 @@ def load_ldlc_medication_effect(key: str, location: str) -> pd.DataFrame:
     return df
 
 
+def load_bmi_exposure(key: str, location: str) -> pd.DataFrame:
+    entity = get_entity(key)
+    location_id = utility_data.get_location_id(location)
+
+    data = get_draws(
+        gbd_id_type="modelable_entity_id",
+        gbd_id=data_values.BMI_MEAN_ME_ID,
+        source=SOURCES.EPI,
+        location_id=location_id,
+        sex_id=SEX.MALE + SEX.FEMALE,
+        gbd_round_id=GBD_2020_ROUND_ID,
+        decomp_step="usa_re",
+        status="best",
+    )
+
+    # core.get_data processing
+    data = data[data.measure_id == MEASURES["Continuous"]]
+    data = data.drop(labels=["modelable_entity_id"], axis="columns")
+    data = vi_utils.filter_data_by_restrictions(
+        data, entity, "outer", utility_data.get_age_group_ids()
+    )
+    data = vi_utils.normalize(data, fill_value=0)
+    data["parameter"] = "continuous"
+    data = data.filter(DEMOGRAPHIC_COLUMNS + DRAW_COLUMNS + ["parameter"])
+    data = vi_utils.reshape(data, value_cols=DRAW_COLUMNS)
+
+    # replace GBD conventions with vivarium conventions
```

Comment:
this block will go into its own function in future PRs and I'll update BMI mean and sd to use it as well

Reply:
I was going to suggest that. I'd actually recommend doing it as part of this PR b/c it makes it far easier to review